.env.dist

```
@@ -1,2 +1,13 @@
 SOLIDINVOICE_ENV=dev
 SOLIDINVOICE_DEBUG=1
+
+# MCP session store: file | memory | cache | framework
+# - file: on-disk (default, single-node deployments)
+# - memory: in-process (dev only; resets on restart)
+# - cache: PSR-16 cache pool, configurable via SOLIDINVOICE_MCP_SESSION_CACHE_POOL
+#   (point to a Redis-backed pool for multi-node)
+# - framework: Symfony session.handler (shares the app's session storage)
+SOLIDINVOICE_MCP_SESSION_STORE=file
+SOLIDINVOICE_MCP_SESSION_CACHE_POOL=cache.mcp.sessions
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOLIDINVOICE_MCP_SESSION_CACHE_POOL key should go before the SOLIDINVOICE_MCP_SESSION_STORE key

(UnorderedKey)